src/diffusers/models/modeling_utils.py

```diff
@@ -537,7 +689,7 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         token = kwargs.pop("token", None)
         revision = kwargs.pop("revision", None)
         torch_dtype = kwargs.pop("torch_dtype", None)
-        subfolder = kwargs.pop("subfolder", None)
+        subfolder = kwargs.pop("subfolder", "")
```

Comment:
Pretty harmless change.

Reply:
In `configuration_utils.py`, `subfolder` default is still `None`.

Reply:
Resolved.